#########################Automated tests trace#########################

self = <MEGS.Tests.CRUD.test_entity_crud_action.TestEntityBase object at 0x041BD350>
generic_entity_test_criteria = <PyTest.InputTestCriteria.InputTestCriteria.GenericEntityTestCriteria object at 0x03557690>
search_api = <Search.Implementation.Contract.search.SearchAPI object at 0x035570B0>

    @pytest.mark.crud
    def test_delete_entity(self, generic_entity_test_criteria, search_api):
        if isinstance(generic_entity_test_criteria, GenericEntityTestCriteria):
            entity = generic_entity_test_criteria.api_object
            entity.es_client = self.es_client
            _primary_key = self.es_client.edge_metadata[entity.entity_name].get_entity_primary_key_field()
    
            ##Create entity
            #http_response = list(entity.create_entity(params={}))
            #obj_id = json.loads(http_response[1])[_primary_key]
            if not self.get_from_local_entity_storage(entity.entity_name):
                #Create entity
                http_response = list(entity.create_entity(params={}))
>               obj_id = json.loads(http_response[1])[_primary_key]

C:\EdgeDeployment\FunctionalTests\src\MEGS\Tests\CRUD\test_entity_crud_action.py:290: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

s = 'Bad input data: Principal team (Id=b0089655-ad56-e311-a147-00155d280475, type=9) is missing prvReadConnection privilege (Id=919f6df3-97bf-4897-82f1-9d67749eab3a)'
encoding = None, cls = None, object_hook = None, parse_float = None
parse_int = None, parse_constant = None, object_pairs_hook = None, kw = {}

    def loads(s, encoding=None, cls=None, object_hook=None, parse_float=None,
            parse_int=None, parse_constant=None, object_pairs_hook=None, **kw):
        """Deserialize ``s`` (a ``str`` or ``unicode`` instance containing a JSON
        document) to a Python object.
    
        If ``s`` is a ``str`` instance and is encoded with an ASCII based encoding
        other than utf-8 (e.g. latin-1) then an appropriate ``encoding`` name
        must be specified. Encodings that are not ASCII based (such as UCS-2)
        are not allowed and should be decoded to ``unicode`` first.
    
        ``object_hook`` is an optional function that will be called with the
        result of any object literal decode (a ``dict``). The return value of
        ``object_hook`` will be used instead of the ``dict``. This feature
        can be used to implement custom decoders (e.g. JSON-RPC class hinting).
    
        ``object_pairs_hook`` is an optional function that will be called with the
        result of any object literal decoded with an ordered list of pairs.  The
        return value of ``object_pairs_hook`` will be used instead of the ``dict``.
        This feature can be used to implement custom decoders that rely on the
        order that the key and value pairs are decoded (for example,
        collections.OrderedDict will remember the order of insertion). If
        ``object_hook`` is also defined, the ``object_pairs_hook`` takes priority.
    
        ``parse_float``, if specified, will be called with the string
        of every JSON float to be decoded. By default this is equivalent to
        float(num_str). This can be used to use another datatype or parser
        for JSON floats (e.g. decimal.Decimal).
    
        ``parse_int``, if specified, will be called with the string
        of every JSON int to be decoded. By default this is equivalent to
        int(num_str). This can be used to use another datatype or parser
        for JSON integers (e.g. float).
    
        ``parse_constant``, if specified, will be called with one of the
        following strings: -Infinity, Infinity, NaN, null, true, false.
        This can be used to raise an exception if invalid JSON numbers
        are encountered.
    
        To use a custom ``JSONDecoder`` subclass, specify it with the ``cls``
        kwarg; otherwise ``JSONDecoder`` is used.
    
        """
        if (cls is None and encoding is None and object_hook is None and
                parse_int is None and parse_float is None and
                parse_constant is None and object_pairs_hook is None and not kw):
>           return _default_decoder.decode(s)

C:\EdgeDeployment\python27\lib\json\__init__.py:338: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <json.decoder.JSONDecoder object at 0x02B533B0>
s = 'Bad input data: Principal team (Id=b0089655-ad56-e311-a147-00155d280475, type=9) is missing prvReadConnection privilege (Id=919f6df3-97bf-4897-82f1-9d67749eab3a)'
_w = <built-in method match of _sre.SRE_Pattern object at 0x029D7638>

    def decode(self, s, _w=WHITESPACE.match):
        """Return the Python representation of ``s`` (a ``str`` or ``unicode``
            instance containing a JSON document)
    
            """
>       obj, end = self.raw_decode(s, idx=_w(s, 0).end())

C:\EdgeDeployment\python27\lib\json\decoder.py:365: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <json.decoder.JSONDecoder object at 0x02B533B0>
s = 'Bad input data: Principal team (Id=b0089655-ad56-e311-a147-00155d280475, type=9) is missing prvReadConnection privilege (Id=919f6df3-97bf-4897-82f1-9d67749eab3a)'
idx = 0

    def raw_decode(self, s, idx=0):
        """Decode a JSON document from ``s`` (a ``str`` or ``unicode``
            beginning with a JSON document) and return a 2-tuple of the Python
            representation and the index in ``s`` where the document ended.
    
            This can be used to decode a JSON document from a string that may
            have extraneous data at the end.
    
            """
        try:
            obj, end = self.scan_once(s, idx)
        except StopIteration:
>           raise ValueError("No JSON object could be decoded")
E           ValueError: No JSON object could be decoded

C:\EdgeDeployment\python27\lib\json\decoder.py:383: ValueError
-------------------------------- Captured log ---------------------------------
requests_api.py             76 INFO     Sending HTTP request...
requests_api.py             83 DEBUG    Method type: GET
requests_api.py             94 DEBUG    Url: http://localhost/xre2113qa.nightly/api/authentication
requests_api.py             95 DEBUG    Headers: {'Authorization-Token': u'6569219054d540fb97925f7d0b93ffb3', 'Accept-Language': 'en-US,en;q=0.8', 'Content-Type': 'application/json', 'Accept-Encoding': 'deflate, gzip'}
connectionpool.py          176 INFO     Starting new HTTP connection (1): localhost
connectionpool.py          344 DEBUG    "GET /xre2113qa.nightly/api/authentication HTTP/1.1" 200 2097
requests_api.py            140 INFO     Response recieved...
requests_api.py            141 INFO     Response status: 200
requests_api.py            142 INFO     Response reason: OK
requests_api.py            143 INFO     Response time: 0:00:00.188000
ConnectionBase.py          118 INFO     Get connection roles...
ConnectionBase.py          119 INFO     Record 1 logical name: contact
ConnectionBase.py          120 INFO     Record 2 logical name: account
EntityBase.py              132 INFO     Sending GET request..
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/connection HTTP/1.1" 200 5244
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/activitypointer HTTP/1.1" 200 9299
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/connection_1 HTTP/1.1" 404 104
elasticsearch_client.py     60 WARNING  ValueWithVersion`1 entity has not been found in index xre2113qa.nightly.settings
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/connection_1/_count HTTP/1.1" 200 62
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/_search HTTP/1.1" 200 1603
requests_api.py             76 INFO     Sending HTTP request...
requests_api.py             83 DEBUG    Method type: GET
requests_api.py             94 DEBUG    Url: http://localhost/xre2113qa.nightly/api/ConnectionRole//
requests_api.py             95 DEBUG    Headers: {'Authorization-Token': u'6569219054d540fb97925f7d0b93ffb3', 'Accept-Language': 'en-US,en;q=0.8', 'Content-Type': 'application/json', 'Accept-Encoding': 'deflate, gzip'}
requests_api.py            127 DEBUG    Request params: record1objecttypecode=contact&record2objecttypecode=account
connectionpool.py          176 INFO     Starting new HTTP connection (1): localhost
connectionpool.py          344 DEBUG    "GET /xre2113qa.nightly/api/ConnectionRole//?record1objecttypecode=contact&record2objecttypecode=account HTTP/1.1" 200 1835
requests_api.py            140 INFO     Response recieved...
requests_api.py            141 INFO     Response status: 200
requests_api.py            142 INFO     Response reason: OK
requests_api.py            143 INFO     Response time: 0:00:04.277000
EntityBase.py              272 DEBUG    Creating request body...
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/account HTTP/1.1" 200 10576
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/activitypointer HTTP/1.1" 200 9299
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/account_1 HTTP/1.1" 404 101
elasticsearch_client.py     60 WARNING  ValueWithVersion`1 entity has not been found in index xre2113qa.nightly.settings
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/account_1/_count HTTP/1.1" 200 60
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/_search HTTP/1.1" 200 1757
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/contact HTTP/1.1" 200 15916
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/activitypointer HTTP/1.1" 200 9299
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/contact_1 HTTP/1.1" 404 101
elasticsearch_client.py     60 WARNING  ValueWithVersion`1 entity has not been found in index xre2113qa.nightly.settings
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/contact_1/_count HTTP/1.1" 200 61
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/_search HTTP/1.1" 200 1651
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/businessunit HTTP/1.1" 404 104
elasticsearch_client.py     60 WARNING  ValueWithVersion`1 entity has not been found in index xre2113qa.nightly.settings
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/activitypointer HTTP/1.1" 200 9299
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/businessunit HTTP/1.1" 404 104
elasticsearch_client.py     60 WARNING  ValueWithVersion`1 entity has not been found in index xre2113qa.nightly.settings
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/businessunit/_count HTTP/1.1" 200 59
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/_search HTTP/1.1" 200 122
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/team HTTP/1.1" 404 96
elasticsearch_client.py     60 WARNING  ValueWithVersion`1 entity has not been found in index xre2113qa.nightly.settings
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/activitypointer HTTP/1.1" 200 9299
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/team HTTP/1.1" 404 96
elasticsearch_client.py     60 WARNING  ValueWithVersion`1 entity has not been found in index xre2113qa.nightly.settings
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/team/_count HTTP/1.1" 200 59
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/_search HTTP/1.1" 200 123
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/opportunity HTTP/1.1" 200 20477
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/activitypointer HTTP/1.1" 200 9299
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/opportunity_1 HTTP/1.1" 404 105
elasticsearch_client.py     60 WARNING  ValueWithVersion`1 entity has not been found in index xre2113qa.nightly.settings
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/opportunity_1/_count HTTP/1.1" 200 61
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/_search HTTP/1.1" 200 1405
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/connectionrole HTTP/1.1" 200 2218
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/activitypointer HTTP/1.1" 200 9299
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/connectionrole_1 HTTP/1.1" 404 108
elasticsearch_client.py     60 WARNING  ValueWithVersion`1 entity has not been found in index xre2113qa.nightly.settings
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/connectionrole_1/_count HTTP/1.1" 200 60
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/_search HTTP/1.1" 200 650
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/goal HTTP/1.1" 404 96
elasticsearch_client.py     60 WARNING  ValueWithVersion`1 entity has not been found in index xre2113qa.nightly.settings
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/activitypointer HTTP/1.1" 200 9299
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/goal HTTP/1.1" 404 96
elasticsearch_client.py     60 WARNING  ValueWithVersion`1 entity has not been found in index xre2113qa.nightly.settings
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/goal/_count HTTP/1.1" 200 59
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/_search HTTP/1.1" 200 124
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/connectionrole HTTP/1.1" 200 2218
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/activitypointer HTTP/1.1" 200 9299
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/connectionrole_1 HTTP/1.1" 404 108
elasticsearch_client.py     60 WARNING  ValueWithVersion`1 entity has not been found in index xre2113qa.nightly.settings
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/connectionrole_1/_count HTTP/1.1" 200 60
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/_search HTTP/1.1" 200 626
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/connection HTTP/1.1" 200 5244
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/activitypointer HTTP/1.1" 200 9299
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/connection_1 HTTP/1.1" 404 104
elasticsearch_client.py     60 WARNING  ValueWithVersion`1 entity has not been found in index xre2113qa.nightly.settings
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/connection_1/_count HTTP/1.1" 200 62
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/_search HTTP/1.1" 200 1568
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/transactioncurrency HTTP/1.1" 200 2229
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/activitypointer HTTP/1.1" 200 9299
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/transactioncurrency_1 HTTP/1.1" 404 113
elasticsearch_client.py     60 WARNING  ValueWithVersion`1 entity has not been found in index xre2113qa.nightly.settings
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/transactioncurrency_1/_count HTTP/1.1" 200 59
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/_search HTTP/1.1" 200 654
EntityBase.py              383 DEBUG    Request body created:
EntityBase.py              384 DEBUG    {"relatedconnectionid": {"logicalname": "connection", "id": "e46b16ad-b561-e311-9471-00155d280475"}, "awx_companyid": {"logicalname": "account", "id": "f5c5dc82-6d92-e311-a5b4-00155d28de35"}, "description": "TEST Chinese / \u00e9\u00a6\u0096\u00e9\u00a1\u00b5", "awx_primarycontactid": {"logicalname": "contact", "id": "91af2e09-8595-e311-aa29-00155d28de35"}, "awx_businessunit": "TEST French / Fran\u00c3\u00a7ais", "record1roleid": {"logicalname": "connectionrole", "id": "55745886-6b14-e111-a170-000c290b0e27"}, "transactioncurrencyid": {"logicalname": "transactioncurrency", "id": "e0a4c3a9-ad56-e311-a147-00155d280475"}, "record1id": {"logicalname": "opportunity", "id": "c0638b6e-8371-e311-bb29-00155d280475"}, "awx_access": "Business Unit", "awx_contributorrole": "TEST Thai / \u00e0\u00b8\u00ab\u00e0\u00b8\u0099\u00e0\u00b9\u0089\u00e0\u00b8\u00b2\u00e0\u00b8\u00ab\u00e0\u00b8\u00a5\u00e0\u00b8\u00b1\u00e0\u00b8\u0081", "type": "connection", "record2roleid": {"logicalname": "connectionrole", "id": "eda69fc6-0b5f-44fb-b584-7dfeb8a925af"}, "name": "TEST Thai / \u00e0\u00b8\u00ab\u00e0\u00b8\u0099\u00e0\u00b9\u0089\u00e0\u00b8\u00b2\u00e0\u00b8\u00ab\u00e0\u00b8\u00a5\u00e0\u00b8\u00b1\u00e0\u00b8\u0081"}
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/contact HTTP/1.1" 200 15916
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/activitypointer HTTP/1.1" 200 9299
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/contact_1 HTTP/1.1" 404 101
elasticsearch_client.py     60 WARNING  ValueWithVersion`1 entity has not been found in index xre2113qa.nightly.settings
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/contact_1/_count HTTP/1.1" 200 61
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/_search HTTP/1.1" 200 1552
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/account HTTP/1.1" 200 10576
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/activitypointer HTTP/1.1" 200 9299
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly.settings/ValueWithVersion%601/account_1 HTTP/1.1" 404 101
elasticsearch_client.py     60 WARNING  ValueWithVersion`1 entity has not been found in index xre2113qa.nightly.settings
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/account_1/_count HTTP/1.1" 200 60
connectionpool.py          296 DEBUG    "GET /xre2113qa.nightly/_search HTTP/1.1" 200 1410
requests_api.py             76 INFO     Sending HTTP request...
requests_api.py             83 DEBUG    Method type: POST
requests_api.py             94 DEBUG    Url: http://localhost/xre2113qa.nightly/api/connection//
requests_api.py             95 DEBUG    Headers: {'Authorization-Token': u'6569219054d540fb97925f7d0b93ffb3', 'Accept-Language': 'en-US,en;q=0.8', 'Content-Type': 'application/json', 'Accept-Encoding': 'deflate, gzip'}
requests_api.py            104 DEBUG    Request body: {"record1id": {"logicalname": "contact", "id": "36f252b1-7e95-e311-aa29-00155d28de35"}, "awx_contributorrole": "TEST Thai / \u00e0\u00b8\u00ab\u00e0\u00b8\u0099\u00e0\u00b9\u0089\u00e0\u00b8\u00b2\u00e0\u00b8\u00ab\u00e0\u00b8\u00a5\u00e0\u00b8\u00b1\u00e0\u00b8\u0081", "name": "TEST Thai / \u00e0\u00b8\u00ab\u00e0\u00b8\u0099\u00e0\u00b9\u0089\u00e0\u00b8\u00b2\u00e0\u00b8\u00ab\u00e0\u00b8\u00a5\u00e0\u00b8\u00b1\u00e0\u00b8\u0081", "record2roleid": {"logicalname": "connectionrole", "id": "2b55bb79-880c-e111-a170-000c290b0e27", "name": "Vendor"}, "record2id": {"logicalname": "account", "id": "6713dcaf-f594-e311-aa29-00155d28de35"}, "awx_access": "Business Unit", "type": "connection", "ismaster": false, "record1roleid": {"logicalname": "connectionrole", "id": "1f2fb088-880c-e111-a170-000c290b0e27", "name": "Client"}}
connectionpool.py          176 INFO     Starting new HTTP connection (1): localhost
connectionpool.py          344 DEBUG    "POST /xre2113qa.nightly/api/connection// HTTP/1.1" 400 161
requests_api.py            140 INFO     Response recieved...
requests_api.py            141 INFO     Response status: 400
requests_api.py            142 INFO     Response reason: Bad Request
requests_api.py            143 INFO     Response time: 0:00:00.320000
#########################Server error trace#########################
{"index":[],"megs":[{"TimeStamp":"2014-02-18T08:47:08.0026761Z","Message":"FaultException: ","ErrorLevel":"ERROR","Longdate":"2014-02-18 02:47:04.8583","ThreadId":"10 ","Logger":"AscendixRE.MEGS.ApiControllers.BaseControllers.Dynamic.DynamicEntityController`1[[DetailSchema_1.connection, a6c3f7df403142b8b87f32c28345d74f, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null]]","Exception":"System.ServiceModel.FaultException`1[Microsoft.Xrm.Sdk.OrganizationServiceFault]: Principal team (Id=b0089655-ad56-e311-a147-00155d280475, type=9) is missing prvReadConnection privilege (Id=919f6df3-97bf-4897-82f1-9d67749eab3a) (Fault Detail is equal to Microsoft.Xrm.Sdk.OrganizationServiceFault)."},{"TimeStamp":"2014-02-18T08:47:08.0866551Z","Message":"Inner Fault Message: Principal team (Id=b0089655-ad56-e311-a147-00155d280475, type=9) is missing prvReadConnection privilege (Id=919f6df3-97bf-4897-82f1-9d67749eab3a)","ErrorLevel":"ERROR","Longdate":"2014-02-18 02:47:04.8671","ThreadId":"10 ","Logger":"AscendixRE.MEGS.ApiControllers.BaseControllers.Dynamic.DynamicEntityController`1[[DetailSchema_1.connection, a6c3f7df403142b8b87f32c28345d74f, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null]]","Exception":""},{"TimeStamp":"2014-02-18T08:47:08.4040176Z","Message":"Inner Fault Trace Text: ","ErrorLevel":"ERROR","Longdate":"2014-02-18 02:47:04.8671","ThreadId":"10 ","Logger":"AscendixRE.MEGS.ApiControllers.BaseControllers.Dynamic.DynamicEntityController`1[[DetailSchema_1.connection, a6c3f7df403142b8b87f32c28345d74f, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null]]","Exception":""},{"TimeStamp":"2014-02-18T08:47:08.4792081Z","Message":"Exception: ","ErrorLevel":"ERROR","Longdate":"2014-02-18 02:47:04.8671","ThreadId":"10 ","Logger":"AscendixRE.MEGS.ApiControllers.BaseControllers.Dynamic.DynamicEntityController`1[[DetailSchema_1.connection, a6c3f7df403142b8b87f32c28345d74f, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null]]","Exception":"System.ArgumentException: Bad input data: Principal team (Id=b0089655-ad56-e311-a147-00155d280475, type=9) is missing prvReadConnection privilege (Id=919f6df3-97bf-4897-82f1-9d67749eab3a)\r\n   at AscendixRE.MEGS.ApiControllers.BaseControllers.EntityControllerBase.CheckSenderFault(FaultException ex)\r\n   at AscendixRE.MEGS.ApiControllers.BaseControllers.EntityControllerBase.CreateEntity(Entity sdkEntity)\r\n   at System.Dynamic.UpdateDelegates.UpdateAndExecute3[T0,T1,T2,TRet](CallSite site, T0 arg0, T1 arg1, T2 arg2)\r\n   at CallSite.Target(Closure , CallSite , EntityControllerBase , ICRMEntity , Object )\r\n   at AscendixRE.MEGS.ApiControllers.BaseControllers.EntityControllerBase.Post(Object entity)\r\n   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass13.<GetExecutor>b__c(Object instance, Object[] methodParameters)\r\n   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.Execute(Object instance, Object[] arguments)\r\n   at System.Threading.Tasks.TaskHelpers.RunSynchronously[TResult](Func`1 func, CancellationToken cancellationToken)"}]}
